Source/Public/Invoke-DockerLint.ps1

```diff
@@ -25,7 +25,8 @@ function Invoke-DockerLint {
     }
     $hadoLintImage = 'hadolint/hadolint:v1.17.2'
```

Comment:
I think this should be configurable as a parameter but with a default at least tag.

Reply:
The specific version of the hadolint tool constitutes part of what we have tested when we released a specific version of this module, so I would prefer not to expose the internals to users of the module.

Reply:
We shouldn't lock in users

Should be an parameter with 
`$HadoLintTag = 'v1.17.2'`
```suggestion
    $hadoLintImage = 'hadolint/hadolint:$HadoLintTag '
```